app/views/Home.scala.html

```diff
@@ -141,7 +141,10 @@
                         </span>
                     }
                     <span class="govuk-visually-hidden">@messages("home.agent.visualHidden.clientsName")</span>
+                @*
```

Comment:
pls remove this, I was referring to the fact that we need to find out where about this data  commit from; 
revert last changes related to userName bug, this to be address in the separate PR

Reply:
reverted